connectors/tests/test_byoei.py

```diff
@@ -8,11 +8,11 @@
 import pytest
 
 from connectors.byoc import PipelineSettings
-from connectors.byoei import ElasticServer
+from connectors.byoei import ElasticServer, IndexMissing
 
 
 @pytest.mark.asyncio
-async def test_prepare_index(mock_responses):
+async def test__override_index(mock_responses):
```

Comment:
two `__`, just use one

Reply:
I was hesitant about this cause of `_override_index` :D

Reply:
yeah gotcha. just a nit though, it's usually bad practiove to have `__` in names unless when doing name mangling